Update `Guides/Tablet Purchase` (#6094)
ppy/osu-wiki: wiki/Guides/Tablet_Purchase/en.md

### Huion

> Huion is a tablet company that is the supplier of the osu!tablet. They are best known for being cheaper than Wacom tablets by a large margin but some would say that their tablets feel flimsier and that running a tablet pen over a Huion tablet would feel as if you were on writing paper (in comparison, some say that on Wacom, it is as if you were writing on glass). Their tablet pens either require a battery or needs to be charged, unlike Wacom tablet pens, and their nibs may need to be often replaced than Wacom nibs. Other than that, there really are not any noticeable differences between Walcom and Huion tablets.

Reviewer: ```suggestion
Huion is a tablet company that is the supplier of the osu!tablet. They are best known for being cheaper than Wacom tablets by a large margin but some would say that their tablets feel flimsier and that running a tablet pen over a Huion tablet would feel as if you were on writing paper (in comparison, some say that on Wacom, it is as if you were writing on glass). Their tablet pens either require a battery or needs to be charged, unlike Wacom tablet pens, and their nibs may need to be replaced more often than Wacom nibs. Other than that, there really are not any noticeable differences between Wacom and Huion tablets.
```

![](img/gen1.jpg "First generation Bamboo (Pen and Touch, Touch, and Fun - Bamboo Fun shown)")

> ![](img/gen2.jpg "Second generation Bamboo (Pen and Touch, Pen, Touch, Fun, and Craft - Bamboo Pen and Touch shown)")

Reviewer 2: I'm pretty sure these images are missing in the PR?

Reviewer: it looks like they were there all the time (since 4 years ago, yeah)

Author: Appear just fine on mine and https://github.com/ppy/osu-wiki/tree/master/wiki/Guides/Tablet_Purchase/img
![IMG_20210913_161654](https://user-images.githubusercontent.com/76740201/133058347-0a2b3186-4379-478a-bbb8-ccf661bd8605.jpg)

Reviewer 2: Ah, right. Thought they were new ones.

![](img/gen3.jpg "Third generation Bamboo (Connect, Capture, Create - Connect shown)")

You'll want to stay away from either of the Bamboo Touch tablets, they do not feature pen input, they're just multitouch trackpads for your computer. Frankly, [Apple did it better](https://apple.com/magictrackpad).

The other Bamboo models come in two sizes, basically small and large. For the second generation, Bamboo Touch, Pen, and Pen and Touch were the "small" ones, whereas the Fun and Craft were the "large" ones. For the third generation, the Connect and Capture are the "small", whereas the Create is the "large".

For osu!, you'll probably be looking at getting one of the "small" sized tablets because it's a common strategy to reduce play area so that you don't have to move your arm when you play. However, it's personal preference as to whether or not you want a larger tablet area, and it's especially worth considering the "large" size if you are also into digital art; drawing is much more comfortable on a larger tablet.

> Lastly, CTL-480 tablets can be had for very cheap on second-hand sites like eBay or Amazon, although they are often hard to find, due to the discontinuation of the manufacturing, but if you could find one for a reasonable price, they are definitely worth checking out if you're also going to be doing digital art as well. For osu!, however, the differences between the Intuos and Bamboo lines are negligible, the only real difference is that the Intuos lines have a much wider range of sizes than is available for Bamboo.

Reviewer: what about ctl-470 and other similar models?

> Lastly, CTL-480 tablets can be had for very cheap on second-hand sites like eBay or Amazon, although they are often hard to find, due to the discontinuation of the manufacturing, but if you could find one for a reasonable price, they are definitely worth checking out if you're also going to be doing digital art as well. Beside the Wacom CTL-480, that is commonly used and praised by players. There is also the CTL-470 lineup (such as the 470, 471 and 472) and CTL-4100, which is the latest models from Wacom, offering the same quality, if not better than the CTL-480.

Reviewer: ```suggestion
Lastly, CTL-480 tablets may be sold cheap on second-hand sites like eBay or Amazon, although they are often hard to find due to the discontinuation of the manufacturing. If you could find one for a reasonable price, they are definitely worth checking out, especially if you're also a digital artist. Beside Wacom CTL-480, which is commonly used and praised by players, there is also the CTL-470 lineup (such as the 470, 471 and 472 models), and the latest CTL-4100 series, which offers the same quality, if not better than CTL-480.
```

### Other tablet brands

> Other than Wacom and Huion, there are also more tablet manufacturers that tries to compete with the market, bringing a decent product that targetted at artists, and even osu! players.

Reviewer: ```suggestion
Beside Wacom and Huion, there are other competitors on the market, who manufacture decent tablets targeted at artists, and even osu! players.
```

> Some of those manufacturer and their known tablet are:

Reviewer: ```suggestion
Some of those manufacturers are:
```

- Gaomon (known for their Gaomon S620 tablet).
- XP-Pen (known for their XP-PEN G640 Rev A tablet).

> - VEIKK  (known for their VEIKK S640 tablet).

Reviewer: ```suggestion
- VEIKK (known for their VEIKK S640 tablet).
```

> All of those aforementioned tablets have a pretty decent quality. Often being priced much cheaper than Wacom's tablets, making them a really good tablet when you're on budget.

Reviewer: ```suggestion
All of tablets mentioned are of decent quality and often cost less than Wacom's tablets, which makes them good budget options.
```

Reviewer 3: ```suggestion
All of the tablets mentioned are of decent quality and often cost less than Wacom's tablets, which makes them good budget options.
```

> Although, some people still prefer the Wacom's tablets, despite of those alternatives.

Reviewer: this sentence sounds unnecessary

Author: agree, removed it

> In short, Wacom is the recommended brand from most serious osu! players. Cintiq and Intuos tablets are too expensive and give no real advantage over Bamboo for osu!. See the recommend options above for the recommend models. Also try your luck at a used CTL-480 if you're also into art.

Reviewer: I think everything else is excessive; besides, there's a section with the recommendations right below
```suggestion
In short, Wacom is recommended by most serious osu! players. Cintiq and Intuos tablets are too expensive and give no real advantage over Bamboo for osu!.
```

Author: got it

## Recommendations

Here are some links to purchase the latest models.

> - [Amazon - One by Wacom Tablet (small version) (CTL472)](https://www.amazon.com/Wacom-Graphic-Drawing-Tablet-Beginners/dp/B07S1RR3FR)

Reviewer: can cut it even further
```suggestion
- [Amazon - One by Wacom Tablet (small version) (CTL472)](https://www.amazon.com/dp/B07S1RR3FR)
```

Author: thanks

![](https://puu.sh/dlQH "Second generation Bamboo (Pen and Touch, Pen, Touch, Fun, and Craft - Bamboo Pen and Touch shown)")

> ![](https://wowcake.s-ul.eu/PgLyCagy "Third generation Bamboo (Connect, Capture, Create - Connect shown)")

Reviewer 2: image should be included in PR. we don't support external hosting like this.

Author: Right, i'll change it to use the images that are already provided here.

### Other tablet brands

Other than Wacom and Huion, there are also more tablet manufacturers that tries to compete with the market, bringing a decent product that targetted at artists, and even osu! players.

Some of those manufacturer and their known tablet are:

- Gaomon (known for their Gaomon S620 tablet).
- XP-Pen (known for their XP-PEN G640 Rev A tablet).
- VEIKK  (known for their VEIKK S640 tablet).

All of those aforementioned tablets have a pretty decent quality. Often being priced much cheaper than Wacom's tablets, making them a really good tablet when you're on budget.

Although, some people still prefer the Wacom's tablets, despite of those alternatives.

In short, Wacom is the recommended brand from most serious osu! players. Cintiq and Intuos tablets are too expensive and give no real advantage over Bamboo for osu!. See the recommend options above for the recommend models. Also try your luck at a used CTL-480 if you're also into art.

## Recommendations

Here are some links to purchase the latest models.

> - [Amazon - One by Wacom Tablet (small version) (CTL472)](https://www.amazon.com/dp/B07S1RR3FR/ref=cm_sw_r_awdo_navT_g_6QR3RY32BE2C0WQ8B1KG)

Reviewer 2: I hope this isn't intentional malice, but you are attempting to include a referral code in this link.

Author: my bad, didn't realize that the link includes referral code
